docs/web-servers/caddy/how-to-install-and-configure-caddy-on-debian-10/index.md

```diff
@@ -0,0 +1,148 @@
+
+1.  Update your system:
+
+        sudo apt-get update && sudo apt-get upgrade
+
+## Install Caddy
+
+1. Install Caddy. This will install Caddy version 1.0.4. along with the `hook.service` [plugin](https://github.com/hacdias/caddy-service), which gives you access to a systemd unit file that you can use to manage Caddy as a systemd service. See their [downloads page](https://caddyserver.com/v1/download) for more information on available Caddy versions.
+
+        curl https://getcaddy.com | bash -s personal hook.service
+
+    Caddy will be installed to your `/usr/local/bin/caddy` directory.
+
+    {{< note >}}
+To learn about Caddy licensing, please read their [blog post on the topic](https://caddyserver.com/v1/blog/announcing-caddy-1_0-caddy-2-caddy-enterprise). In 2017, commercial use of Caddy and their binaries required a license, however, they have recently updated their licensing and commercial licenses are no longer required for their use.
+    {{</ note >}}
+
+## Add Web Content
+
+In this section, you will create the necessary directories to host your website files, set their correct permissions, and add a basic index file to your example site.
+
+{{< note >}}
+Throughout this section, replace all instances of `example.com` with your own domain.
+{{</ note >}}
+
+1.  Set up a *document root* for your website. A document root is the directory where your website files are stored.
+
+        sudo mkdir -p /var/www/example.com
+
+1. Change your site's document root to be owned by the `www-data` user.
+
+        sudo chown -R www-data:www-data /var/www/example.com
+
+1. Create a test index page for your site. Replace `example.com` with your own domain.
+
+        sudo touch /var/www/example.com/index.html
+
+1. Add the example `html` to your site's index.
+
+        sudo echo '<!doctype html><head><title>Caddy Test Page</title></head><body><h1>Hello, World!</h1></body></html>' | sudo tee /var/www/example.com/index.html
+
+## Configure the Caddyfile
+
+Now that you have your website's document root set up with example content, you are ready to configure Caddy to serve your website files to the internet. This section will create a basic Caddy configuration, which will [automatically enable HTTPS using Let's Encrypt](https://caddyserver.com/v1/).
+
+1. Create a directory to store Caddy's configuration files:
+
+        sudo mkdir -p /etc/caddy
+
+1. Update the directory's owner to be the web server user, `www-data`.
+
+        sudo chown -R www-data:www-data /etc/caddy
+
+1. Using the text editor of your choice, create and edit the [Caddyfile](https://caddyserver.com/docs/caddyfile-tutorial) to serve your example site. The Caddyfile is Caddy's main configuration file. Replace `example.com` with your own domain.
+
+      {{< file "/etc/caddy/Caddyfile" >}}
+example.com {
+    root /var/www/example.com
+}
+      {{</ file >}}
+
+1. Install Caddy as a systemd service and tell it where to find your Caddyfile:
```

Comment:
Do we need to specify a location for the Caddyfile here? Looks like we do this in step 7.

Reply:
Hmm - this is a mistake - I'll fix - left over from when I was rearranging things - thanks!